src/masternodes/masternodes.cpp

```diff
@@ -76,15 +76,33 @@ CAmount GetTokenCreationFee(int)
     return Params().GetConsensus().token.creationFee;
 }
 
-CAmount GetPropsCreationFee(int, CPropType prop)
+CAmount GetPropsCreationFee(int, const CCustomCSView& view, CPropType prop, uint8_t options)
 {
+    auto attributes = view.GetAttributes();
+    if (!attributes) {
```

Comment:
There will always be attributes. You can put `assert(attributes)` here and remove the conditional.